src/components/Infra/InfraEnvForm.tsx

```diff
@@ -121,43 +125,24 @@ const InfraEnvWizard: React.FC = () => {
     };
 
     if (values.enableProxy) {
-      (infraEnv as any).spec.proxy = {
+      infraEnv.spec.proxy = {
         httpProxy: values.httpProxy,
         httpsProxy: values.httpsProxy,
         noProxy: values.noProxy,
       }
     }
 
-    const networks = [];
-
-    values.networks.forEach((n) => {
-      let config;
-      try {
-        config = load(n.config); 
-      } catch {
-        return;
-      }
-      if (n.mac && config?.metadata?.labels) {
-        networks.push({ mac: n.mac, config })
-      }
-    });
-
-    if (networks.length) {
-      (infraEnv as any).spec.nmStateConfigLabelSelector = {
-        matchLabels: networks[0].config.metadata.labels,
-      };
-      await k8sCreate(nmStateModel, networks[0].config);
-    }
-
     return k8sCreate(infraModel, infraEnv);
   }, [infraModel]);
   return (
-    <InfraWizard
-      usedNames={usedNames}
-      onSubmit={onSubmit}
-      onFinish={(values) => history.push(`/k8s/ns/assisted-installer/agent-install.openshift.io~v1beta1~InfraEnv/${values.name}`)}
-      onClose={() => history.push('/k8s/ns/assisted-installer/agent-install.openshift.io~v1beta1~InfraEnv/')}
-    />
+    <div className="infra-env__form">
```

Comment:
It would be nice to achieve the padding using some layout components such as Stack or Grid

Reply:
I agree, but we also cannot use default gutter size because console is using custom 30px AFAIK.